### PR TITLE
document allocated_spend_type for data transfer

### PR DESCRIPTION
**Pull request by mpavlov**
### What does this PR do? What is the motivation?
Document `allocated_spend_type` for data transfer. Also, clarify allocation strategy for data transfer.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 1]
```diff
@@ -141,6 +141,10 @@ ECS tasks that run on Fargate are already fully allocated [in the CUR][103]. CCM
...
 
+Next, Datadog looks at all of the [workload resources][104] running on that node for the day. The cost of the node is allocated to the workload level based on the resources it has used and the length of time it ran. This calculated cost is enriched with all of the workload resource's tags.
+
+**Note**: Only _tags_ from pods and nodes are added to cost metrics. To include labels, enable labels as tags for [nodes][101] and [pods][102].
+
 [Network Performance Monitoring][105] must be enabled on all AWS hosts to allow accurate data transfer cost allocation. If some hosts do not have Network Performance Monitoring enabled, the data transfer costs for these hosts is not allocated and may appear as an `n/a` bucket depending on filter and group-by conditions.
 
 Datadog supports data transfer cost allocation only through the [standard 6 workload resources][104]. If you use custom workload resources their data transfer costs may only be allocated down to the cluster level and not the node/namespace level.
```
Comment (ktaylor19) [inline content/en/cloud_cost_management/container_cost_allocation.md:150]: should we link to https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/ here?

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 2]
```diff
@@ -141,6 +141,10 @@ ECS tasks that run on Fargate are already fully allocated [in the CUR][103]. CCM
 
 For Kubernetes data transfer allocation, a Kubernetes node is joined with its associated data transfer costs from the [CUR][103]. The node's cluster name and all node tags are added to the entire data transfer cost for the node. This allows you to associate cluster-level dimensions with the cost of the data transfer, without considering the pods scheduled to the node.
 
+Next, Datadog looks at all of the [workload resources][104] running on that node for the day. The cost of the node is allocated to the workload level based on the resources it has used and the length of time it ran. This calculated cost is enriched with all of the workload resource's tags.
```
Comment (ktaylor19) [inline content/en/cloud_cost_management/container_cost_allocation.md:144]: do we want to be more specific about "resources" -- e.g. based on the amount of bits etc

File: content/en/cloud_cost_management/container_cost_allocation.md (hunk 1 above)
Comment (mpavlov) [reply content/en/cloud_cost_management/container_cost_allocation.md]: added

File: content/en/cloud_cost_management/container_cost_allocation.md (hunk 2 above)
Comment (mpavlov) [reply content/en/cloud_cost_management/container_cost_allocation.md]: yes, added more detail that we use volume of network traffic to allocate costs

**Review by enemyghost — APPROVED** [review]

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 3]
```diff
@@ -141,15 +141,20 @@ ECS tasks that run on Fargate are already fully allocated [in the CUR][103]. CCM
 
 For Kubernetes data transfer allocation, a Kubernetes node is joined with its associated data transfer costs from the [CUR][103]. The node's cluster name and all node tags are added to the entire data transfer cost for the node. This allows you to associate cluster-level dimensions with the cost of the data transfer, without considering the pods scheduled to the node.
 
+Next, Datadog looks at all of the [workload resources][104] running on that node for the day. The cost of the node is allocated to the workload level based on the volume of network traffic used. This calculated cost is enriched with all of the workload resource's tags.
```
Comment (michaelcretzman) [inline content/en/cloud_cost_management/container_cost_allocation.md:144]:
```suggestion
Next, Datadog examines the daily [workload resources][104] running on that node. The node cost is allocated to the workload level according to network traffic volume usage. This calculated cost is enriched with all of the workload resource's tags.
```

File: content/en/cloud_cost_management/container_cost_allocation.md [hunk 4]
```diff
@@ -141,15 +141,20 @@ ECS tasks that run on Fargate are already fully allocated [in the CUR][103]. CCM
...
+
+**Note**: Only _tags_ from pods and nodes are added to cost metrics. To include labels, enable labels as tags for [nodes][101] and [pods][102].
+
 [Network Performance Monitoring][105] must be enabled on all AWS hosts to allow accurate data transfer cost allocation. If some hosts do not have Network Performance Monitoring enabled, the data transfer costs for these hosts is not allocated and may appear as an `n/a` bucket depending on filter and group-by conditions.
 
-Datadog supports data transfer cost allocation only through the [standard 6 workload resources][104]. If you use custom workload resources their data transfer costs may only be allocated down to the cluster level and not the node/namespace level.
+Datadog supports data transfer cost allocation only through the [standard 6 workload resources][104]. If you use [custom workload resources][106] their data transfer costs may only be allocated down to the cluster level and not the node/namespace level.
```
Comment (michaelcretzman) [inline content/en/cloud_cost_management/container_cost_allocation.md:150]:
```suggestion
Datadog supports data transfer cost allocation using [standard 6 workload resources][104] only. For [custom workload resources][106], data transfer costs can be allocated down to the cluster level only, and not the node/namespace level.
```

Comment (michaelcretzman) [reply content/en/cloud_cost_management/container_cost_allocation.md:150]: Reason for the edit: "may" is often used for either permission or possibility. It's hard for readers to tell. So, for permission, I suggest "can" and for possibility I suggest "might".

**Review by michaelcretzman — CHANGES_REQUESTED** [review]

**Pull request merged**